README.md

```diff
@@ -251,6 +251,7 @@ You can use all of the following options with the standalone version of the <red
   * **path-only**: displays a path in the sidebar navigation item.
   * **id-only**: displays the operation id with a fallback to the path in the sidebar navigation item.
 * `showWebhookVerb` - when set to `true`, shows the HTTP request method for webhooks in operations and in the sidebar.
+* `codeSamplesLanguages` - enables code sample generation for the provided list of languages.
```

Comment:
I wonder why this is needed? Can we not determine this from the content type? (`application/json`, vs `application/xml`, vs `text/csv`)?

Reply:
This is used to allow the user to control for which media types they want code examples to show up rather than showing all code examples for all media types supported (in case the user doesn't want code examples for certain media types to show up by default).